SoftLayer/CLI/image/import.py

```diff
@@ -18,13 +18,38 @@
 @click.option('--os-code',
               default="",
               help="The referenceCode of the operating system software"
-                   " description for the imported VHD")
+                   " description for the imported VHD, ISO, or RAW image")
+@click.option('--ibm-api-key',
+              default="",
+              help="The IBM Cloud API Key with access to IBM Cloud Object "
+                   "Storage instance.")
+@click.option('--root-key-id',
+              default="",
+              help="ID of the root key in Key Protect")
+@click.option('--wrapped-dek',
+              default="",
+              help="Wrapped Decryption Key provided by IBM KeyProtect")
+@click.option('--kp-id',
+              default="",
+              help="ID of the IBM Key Protect Instance")
+@click.option('--cloud-init',
+              default="",
+              help="Specifies if image is cloud init")
```

Comment:
The term, "cloud-init" should have hyphen, as it is the name of a specification. Change to, "Specifies whether image is enabled for cloud initialization using cloud-init".